### PR TITLE
[Feature] added a button for adding and removing friends

### PR DESCRIPTION
**Pull request by Bukii**

### CODE REVIEW

**Comment by TheFern2** [comment]
I tested with my original testing auth profile, and duped two more with diff uuids just for testing purposes.
When the extension is launched, the add profile icon is defaulted to remove icon instead of add.

image

```
query: DELETE FROM "friend" WHERE "userId" = $1 AND "friendsUserId" = $2 -- PARAMETERS: ["5fe1a9c5-d6d5-4cac-89d1-75182b6ae17d","5fe1a9c5-d6d5-4cac-89d1-75182b6ae174"]
query: INSERT INTO "friend"("userId", "friendsUserId") VALUES ($1, $2) -- PARAMETERS: ["5fe1a9c5-d6d5-4cac-89d1-75182b6ae17d","5fe1a9c5-d6d5-4cac-89d1-75182b6ae174"]
```
It looks like the icons are reversed along with functionality. But deleting and removing friends does work in the api. I was able to hit the delete button first which does nothing unless friend data is already there, then add friend, then delete friend.

image

So to summarize:
- every time extension is launched friends show the remove icon even when they are not friends. So is an issue either with ViewStory.svelte or is-friend route not working.

**Comment by Bukii** [comment]
Ok, thank you, I will look into that! :)

**Review by TheFern2 — CHANGES_REQUESTED** [review]
Button status might not be shown with correct status upon clicking on each user, or upon relaunch of extension.

**Comment by Bukii** [comment]
Should be fixed now. Changed the operator from "!==" to "!=".

**Comment by TheFern2** [comment]
I pulled latest commit, and still doing the same thing. Add/Remove icon defaults to remove friend all the time.
Also I just noticed that if you're the user, your profile shouldn't have an add/remove friend for yourself.

**Comment by TheFern2** [comment]
This is tested, and merged. :smiley:
